web/ASC.Web.Common/src/components/Headline/StyledHeadline.js

```diff
@@ -15,7 +15,7 @@ const weight = {
 
 const StyledHeading = styled(Heading)`
   margin: 0;
-  line-height: 56px;
+  line-height: 65px;
```

Comment:
The border-bottom isn't visible on a desktop view on Tablet.

Reply:
fixed